resources/autohotspot/dhcpcd/autohotspot.service

```diff
@@ -0,0 +1,11 @@
+[Unit]
+Description=Automatically generates an wifi hotspot when a valid ssid is not in range
+After=multi-user.target
```

Comment:
Do we need network services available?

Reply:
Was the state before and honstly just copied from the tutorial / script used as base.